src/Writer/WriterInterface.php

```diff
@@ -26,9 +26,10 @@ public function addFilter($filter);
      * Set a message formatter for the writer
      *
      * @param string|Formatter $formatter
+     * @param  array|null $options
      * @return WriterInterface
      */
-    public function setFormatter($formatter);
+    public function setFormatter($formatter, array $options = null);
```

Comment:
Please remove this change. It's a BC break (all implementations would need to be updated, including _userland_ implementations).

The reason PHP 7.2 was raising issues was due to how the `AbstractWriter` is defined, as it adds the additional, optional argument. Any class that extends _that_ class needs to be updated to comply with PHP 7.2, but the interface can remain as-is.